content/applications/websites/ecommerce/products.rst

```diff
@@ -4,266 +4,341 @@
-   installed).
+   - The **Inventory** app must be installed to see the inventory management options.
+   - To display the stock level on your product page, the :guilabel:`Product Type` must be set to
+     :guilabel:`Storable` in the :ref:`product form <products/product-form>`.
 
 Inventory
 ---------
 
-In the :guilabel:`Inventory Defaults` sub-section, you can select the eCommerce selling strategy of
-products:
-
-- :guilabel:`Warehouse`: if you have multiple warehouses, you can define the warehouse associated to
-  your website. If you have multiple websites, you can select a different one per website;
-- :guilabel:`Out-of-Stock (Continue Selling)`: enabling it allows customers to continue placing
-  orders even when the product is **out-of-stock**.
-  Leave it  unchecked to **prevent orders**;
-- :guilabel:`Show Available Qty`: enabling it displays the available quantity left under a specified
-  threshold on the product page. The available quantity is calculated based on the 'On hand'
-  quantity minus the quantity already reserved for outgoing transfers.
-
-.. seealso::
-   :ref:`Allow only selected customers to buy <cart/prevent-sale>`
+In the :guilabel:`Inventory Defaults` sub-section, select your eCommerce selling strategy:
 
-Selling as kit
---------------
-
-If you are selling non-prepackaged kits (i.e., the kits are made of individual products), we
-recommend you read the related documentation to keep track of your stock.
+- :guilabel:`Out-of-Stock`: enable :guilabel:`Continue Selling` to allow customers to continue
+  placing orders even when the product is **out-of-stock**. Leave the box unchecked to
+  **prevent orders**;
+- :guilabel:`Show Available Qty`: displays the available quantity left under a specified threshold
+  on the product page. The available quantity is calculated based on the 'On hand' quantity minus
+  the quantity already reserved for outgoing transfers.
 
-.. seealso::
-   :doc:`../../inventory_and_mrp/manufacturing/advanced_configuration/kit_shipping`
+.. _products/product-comparison:
 
 Product comparison
 ==================
 
-You can enable a **product comparison tool** for your eCommerce by going to
-:menuselection:`Website --> Configuration --> Settings --> Shop - Products`, and ticking
-:guilabel:`Product Comparison Tool`. This tool allows to save products' **specifications** and
-compare them against each other on a single page.
-
-On the product page, scroll down to the :guilabel:`Specifications` section and click
-:guilabel:`Compare`. Repeat the same process for all products you wish to compare. Then, click the
-:guilabel:`Compare` button of the pop-up window at the bottom of the page to reach the comparison
-summary.
+Your visitors can compare products based on their attributes when the :guilabel:`Product Comparison
+Tool` is enabled. You can activate it by going to :menuselection:`Website --> Configuration -->
+Settings --> Shop - Products` section.
```

Comment:
```suggestion
To allow website visitors to compare products based on their attributes, go to :menuselection:`Website --> Configuration --> Settings`, scroll down to the :guilabel:`Shop - Products` section, and enable :guilabel:`Product Comparison Tool`.
```
- non-menu items should not be in menuselection
- to simplify the sentence and avoid repetitions (is enabled/you can activate it) + respecting the order of the steps (go to XXX, find the section, enable the feature) helps make the info more readable and clear